### DISCUSSION TITLE
Executable JavaDoc Code Snippets - Gunnar Morling

### DISCUSSION THREAD

**Discussion by giscus** [gunnarmorling/discussions.morling.dev#289] [Announcements]
# Executable JavaDoc Code Snippets - Gunnar Morling

It has been just a few weeks since the release of Java 17, but the first changes scheduled for Java 18 begin to show up in early access builds.
One feature in particular that excites me as a maintainer of different Java libraries is JEP 413 ("Code Snippets in Java API Documentation").

https://www.morling.dev/blog/executable-javadoc-code-snippets/

**Comment by homberghp** [comment]
Quite nice indeed, however, I am having trouble to get the example working with 18-ea.

**Reply by homberghp** [reply]
Stupid me. Your code works as advertised. In my own concoction I forgot the space between snippet and colon (:).

**Reply by gunnarmorling** [reply]
Cool, happy to hear you got it sorted out.
